airbyte-webapp/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "0.40.30",
   "private": true,
   "engines": {
-    "node": "16.18.1"
+    "node": "16.18.1",
+    "pnpm": "7.26.1"
   },
   "scripts": {
     "prepare": "cd .. && husky install airbyte-webapp/.husky",
-    "prestart": "npm run generate-client",
-    "start": "node -r ./scripts/dev-overwrites.js ./node_modules/.bin/vite",
-    "prestart:cloud": "npm run generate-client",
-    "start:cloud": "AB_ENV=${AB_ENV-frontend-dev} node -r ./scripts/environment.js -r ./scripts/dev-overwrites.js ./node_modules/.bin/vite",
-    "prebuild": "npm run generate-client",
+    "prestart": "pnpm run generate-client",
+    "start": "NODE_OPTIONS='-r ./scripts/dev-overwrites.js' vite",
```

Comment:
:information_source: With `pnpm` the `node_modules/.bin/vite` script is actually a wrapping Bash script not the `js` file of `vite` directly anymore, so we can't call it with `node` anymore, so using `NODE_OPTIONS` to pass down the `-r` parameter to the `node` process that will execute `vite`.